services/proxy/pkg/middleware/oidc_auth.go

```diff
@@ -75,6 +80,10 @@ func (m *OIDCAuthenticator) getClaims(token string, req *http.Request) (map[stri
 		return nil, errors.Wrap(err, "failed to verify access token")
 	}
 
+	if err := aClaims.Valid(); err != nil {
+		return nil, err
+	}
```

Comment:
No need to verify again, because the JWT is Verified in `ParseWithClaims` because `SkipClaimsValidation` is always false.